mev-boost-relay/services/api/constraints.go

```diff
@@ -25,15 +30,15 @@ type SignedConstraints struct {
 type ConstraintsMessage struct {
 	ValidatorIndex uint64        `json:"validator_index"`
 	Slot           uint64        `json:"slot"`
-	Constraints    []*Constraint `ssz-max:"256" json:"constraints"`
+	Constraints    []*Constraint `ssz-max:"1048576" json:"constraints"`
 }
 
 type Constraint struct {
-	Tx    Transaction `ssz-max:"1048576" json:"tx"`
+	Tx    Transaction `ssz-max:"1073741824" json:"tx"`
```

Comment:
Note: this value was wrong, however we only use JSON for this type currently.
We should either remove SSZ definitions for it or start using it.